remove-backup.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+cd /Users/eminmahrt/Developer/nuri-ios
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)